src/styles/components/_card.scss

```diff
@@ -0,0 +1,55 @@
+.product-list {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.product-card {
+  text-align: center;
+  width: 24%;
+  padding: 1rem 0.1rem;
+}
+
+@media screen and (max-width: 800px) {
+  .product-card {
+    width: 50%;
+  }
+}
```

Comment:
As medias querie estão quebrando, fica um espaço a direita no destop e no mobile aparece só um elemento.
```suggestion

.product-list {
  display: flex;
  flex-wrap: wrap;
  justify-content: space-around;
}

.product-card {
  text-align: center;
  width: calc(25% - 16px);
  margin: 16px 0;
}

@media screen and (max-width: 800px) {
  .product-card {
    width: calc(50% - 48px);
  }
}
```

![123](https://user-images.githubusercontent.com/42524424/83986600-bdae2200-a913-11ea-9610-a8ce391ba409.png)

Reply:
> As medias querie estão quebrando, fica um espaço a direita no destop e no mobile aparece só um elemento.
> 
> ![123](https://user-images.githubusercontent.com/42524424/83986600-bdae2200-a913-11ea-9610-a8ce391ba409.png)

Então, nós vimos isso mesmo. Só que achamos que isso deveria ser ajustado quando fizéssemos a Home mesmo. Aqui só renderizamos os produtos mesmo. Acho que ia evitar trabalho duplo. Mas, se for o caso, ajustamos aqui mesmo.